### ISSUE TITLE
Enhancement: Deepgram STT/TTS

### ISSUE THREAD

**Issue by AlexM4H** [LibreChat-AI/LibreChat#3481]
### What features would you like to see added?

What about "Upcoming STT/TTS Enhancements: The Google Cloud STT/TTS and Deepgram services are being planned for future integration."

### More details

I found no issue for this planned enhancement. So I added one. I am interested in the real-time capabiliities of the deepgram API.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by berry-13** [comment]
I've got the Deepgram STT/TTS all set to go, but I need to get PR #3431 merged first. Right now, I’m just using the regular Deepgram API, but we could totally add an option to stream STT and TTS

**Comment by berry-13** [comment]
@AlexM4H if you're interested in Google STT/TTS, create another issue. thanks

**Comment by danny-avila** [comment]
Triage note: the blocker named above cleared long ago, but the implementation never landed.

PR #3431 (the speech services refactor @berry-13 was waiting on) merged on 2024-07-30, the same day as the comment above. The actual Deepgram implementation, PR #4784, was opened 2024-11-24 and closed **without merging** on 2026-04-11. Nothing matching `deepgram` exists anywhere in main today, and the provider strategy maps in `api/server/services/Files/Audio/STTService.js` and `TTSService.js` still register only OpenAI and Azure OpenAI. So this is unimplemented and its one attempt is dead.

One design note for whoever picks it up: #4784 wired Deepgram through the Deepgram SDK rather than the existing generic HTTP provider strategy, which is why it needed new `sdkStrategies` plumbing in both services plus a `getVoices` branch. That choice is worth re-deciding up front, because the realtime streaming @AlexM4H originally asked about is only reachable on the SDK path.

Classification: enhancement, severity low (Deepgram is an optional provider; OpenAI, Azure, ElevenLabs and LocalAI all work), priority low (0 upvotes and a single requester in two years).
